packages/peregrine/lib/talons/CheckoutPage/useCheckoutPage.js

```diff
@@ -56,12 +56,7 @@ export const useCheckoutPage = props => {
 
     const [
         getCheckoutDetails,
-        {
-            data: checkoutData,
-            called: checkoutCalled,
-            client,
-            loading: checkoutLoading
-        }
+        { data: checkoutData, called: checkoutCalled, loading: checkoutLoading }
```

Comment:
Removed `client` here since we can use the `client` from `useApolloClient` which is the same object.